### ISSUE TITLE
USER STORY: Split Datasets

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/breast-cancer-detector#12]
As a **Developer** I can *train, validate and test the models** so that **I can train and evaluate a model's performance**.

**Acceptance Criteria**
* Split the dataset into train, validation and test folders in a 70%/15%/15% split. 
* Ensure the datasets are separate so the test set is unseen to prevent overfitting.
